src/euphorie/client/browser/templates/portlet-my-ras.pt

```diff
@@ -109,8 +109,14 @@
             <p>
               <a class="pat-modal icon-plus-circle"
                  href="${here/absolute_url}/@@new-session.html#document-content"
-                 data-pat-modal="class: panel small"
-              >${view/label_start_session}</a>
```

Comment:
Should `label_start_session` be deprecated/removed?

Reply:
Deprecated.

Reply:
I am not really sure of this change.
Why do you want to remove this customization entry point?
AFAIU, this is used to adjust the link text without overriding the full template of the portlet.
As you mentioned, this is exploited in some customer projects.
In addition, what you are doing does not change anything **except** that `view/label_start_session` returns the text **capitalized** (which should probably be done in CSS, but I do not dare to enter in the merit).

Reply:
Good point, I will double check where this is used and why.

Reply:
This certainly needs more attention. I am withdrawing that change from this PR for the moment and will make a separate one only for this once I have sorted it out. I pushed a cleaned version where the deprecation is removed and the old translation is back. Just the medium modal size is still in.